### ISSUE TITLE
textLength field missing for SVG element Dom.text()

### ISSUE THREAD

**Issue by dave-doty** [Workiva/over_react#342]
> * Issue Type: [BUG]
> * `over_react` Version(s): `2.5.2`

The SVG text element should have `textLength` as an available field (documented here: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/textLength), but it does not.

The following should work but fails to compile:

```
ReactElement text_elt = (Dom.text()..textLength='50')('hello');
```

The workaround is to assigned the property by a `String` key in the intermediate object of type `SvgProps` :

```
SvgProps text_props = Dom.text();
text_props.addProp('textLength', '50');
ReactElement text_elt = text_props('hello');
```

---

> __FYI:__ @greglittlefield-wf @aaronlademann-wf @kealjones-wk @evanweible-wf @maxwellpeterson-wf

**Comment by dave-doty** [comment]
There's a similar issue with the "startOffset" attribute of SVG textPath:

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/startOffset

**Comment by greglittlefield-wf** [comment]
Thanks for reporting this! Looks like we're quite a bit behind on some of our attributes.

Once we're finished with the React 16 upgrade (over_react 3.0.0), we'll update them to match this list: https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes

**Comment by aaronlademann-wf** [comment]
@dave-doty you will find those attributes available in version 3.1.0 - which will be released today: https://github.com/Workiva/over_react/pull/416

**Comment by dave-doty** [comment]
@aaronlademann-wf Great!

Are you sure all the attributes are there? `textLength` seems to work now, but not `y`, for instance. If you try

```
SvgProps props = Dom.text()..y = '0';
```

it generates the error message

```
Error: The setter 'y' isn't defined for the class 'SvgProps'.
 - 'SvgProps' is from 'package:over_react/src/component/dom_components.dart' ('packages/over_react/src/component/dom_components.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'y'.
    SvgProps props = Dom.text()..y = '0';
```
